src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfaceProgress.cs

```diff
@@ -28,10 +29,19 @@ class ConsoleHostUserInterface : System.Management.Automation.Host.PSHostUserInt
             // destroy the data structures representing outstanding progress records
             // take down and destroy the progress display
 
+            if (_progPaneUpdateTimer != null)
+            {
+                // Stop update a progress pane and destroy timer
+                _progPaneUpdateTimer.Dispose();
+                _progPaneUpdateTimer = null;
+            }
+            progPaneUpdateFlag = false;
```

Comment:
Quoted from [MSDN](https://msdn.microsoft.com/en-us/library/zb0225y6(v=vs.110).aspx):
> Callbacks can occur after the Dispose() method overload has been called, because the timer queues callbacks for execution by thread pool threads. 

That means line 38 here doesn't guarantee the flag is set to false, because the timer call back may still run and set it to true. I think we don't need the line 38, because
1. the flag is purely for optimization, calling `Show` one time more sometimes won't affect the correctness, and thus it doesn't have to be really accurate.
2. when we create a new timer for the next `Write-Progress` call, we will set the flag to true anyway.